Project 1 Day 2/application.py

```diff
@@ -54,10 +54,16 @@ def validate_password(self, password):
 def index():
     return render_template("index.html")
 
+#TASK - 5 INSERTING RECORDSINTO DATABASE
 @app.route("/register" , methods=['POST'])
 def register():
-    # print(request.form['Username'])
-    result = request.form
-    # print(result.items())
-    return render_template("result.html", result = result.items())
+	indata = Dataentry(request.form['Username'], request.form['Email'], request.form['password'], datetime.now())
```

Comment:
datetime can be part of the constructor as it is always going to  be datetime.now()